domain/src/main/kotlin/no/nav/su/se/bakover/domain/VedtakInnhold.kt

```diff
@@ -51,7 +87,55 @@ sealed class VedtakInnhold {
         override val fradragSum: Int,
         val avslagsgrunn: Avslagsgrunn,
         val halvGrunnbeløp: Int,
-    ) : VedtakInnhold()
+    ) : VedtakInnhold() {
+        companion object {
+            fun lagAvslagsvedtak(person: Person, behandling: Behandling) =
+                Avslagsvedtak(
+                    dato = LocalDate.now().format(DateTimeFormatter.ofPattern("dd.MM.yyyy")),
+                    fødselsnummer = behandling.søknad.søknadInnhold.personopplysninger.fnr,
+                    fornavn = person.navn.fornavn,
+                    etternavn = person.navn.etternavn,
+                    adresse = person.adresse?.adressenavn,
+                    bruksenhet = person.adresse?.bruksenhet,
+                    husnummer = person.adresse?.husnummer,
+                    postnummer = person.adresse?.poststed?.postnummer,
+                    poststed = person.adresse?.poststed?.poststed!!,
+                    satsbeløp = behandling.beregning()?.månedsberegninger?.firstOrNull()?.satsBeløp ?: 0,
+                    fradragSum = behandling.beregning()?.fradrag?.toFradragPerMåned()?.sumBy { fradrag -> fradrag.beløp } ?: 0,
+                    avslagsgrunn = avslagsgrunnForBehandling(behandling)!!,
+                    halvGrunnbeløp = Grunnbeløp.`0,5G`.fraDato(LocalDate.now()).toInt()
+                )
+        }
+    }
+
+    companion object {
+        fun lagVedtaksinnhold(person: Person, behandling: Behandling): VedtakInnhold =
+            when {
+                erAvslått(behandling) -> lagAvslagsvedtak(person, behandling)
+                erInnvilget(behandling) -> lagInnvilgelsesvedtak(person, behandling)
+                else -> throw java.lang.RuntimeException("Kan ikke lage vedtaksinnhold for behandling som ikke er avslått/innvilget")
+            }
+
+        private fun erInnvilget(behandling: Behandling): Boolean {
```

Comment:
Føles som en funksjon som tilhører Behandling

Reply:
Denne var bare flyttet fra Brevservice til en egen fil slik at det ikke blir for mye ting som ikke har direkte med brevservice å gjøre.